routes/todo.go

```diff
@@ -0,0 +1,63 @@
+package routes
+
+import (
+	"errors"
+	"fmt"
+	"github.com/gofiber/fiber/v2"
+	"strconv"
+)
+
+func createErrorMap(err error) fiber.Map {
+	return fiber.Map{
+		"success": false,
+		"error":   err.Error(),
+	}
+}
+func (h handler) GetAll(c *fiber.Ctx) error {
+	todoList := h.DB.FetchAll()
+
+	return c.Status(fiber.StatusOK).JSON(fiber.Map{
+		"success": true,
+		"data":    todoList,
+	})
+
+}
+
+func (h handler) GetTodo(c *fiber.Ctx) error {
+	idParam := c.Params("id")
+	id, errConvert := strconv.Atoi(idParam)
+	if errConvert != nil {
+		err := fmt.Errorf("ID: %v is not from a valid type", idParam)
+		return c.Status(fiber.StatusBadRequest).JSON(createErrorMap(err))
+	}
+
+	todoText, err := h.DB.FetchTodo(id)
+	if err != nil {
+		return c.Status(fiber.StatusNotFound).JSON(createErrorMap(err))
+	}
+
+	return c.Status(fiber.StatusOK).JSON(fiber.Map{
+		"success": true,
+		"id":      id,
+		"todo":    todoText,
+	})
+
+}
+
+func (h handler) CreateTodo(c *fiber.Ctx) error {
+	todoText := c.FormValue("todo", "")
```

Comment:
You cannot use form data for json

<img width="1501" alt="Screenshot 2023-07-08 at 14 18 11" src="https://github.com/kevinpita/go-todo/assets/84069271/d9056011-faa6-455b-9685-f992e0959322">

Reply:
Fixed 18b05bb6bc084abeb3f8d0be1df5d5ace56938da